src/hooks/useValidation.js

```diff
@@ -17,6 +18,11 @@ const useValidation = (credentials, userSignup, clubSignup) => {
       error: true,
       message: "Password must be at least 6 characters long",
     });
+  } else if (!passwordRegex.test(credentials.password)) {
+    errors.push({
+      error: true,
+      message: "Password must contain at leat 1 digit",
```

Comment:
```suggestion
      message: "Password must contain at leat 1 digit",
```
message: "Password must contain at least 1 digit",